frontend/src/exams/capture-ind-ita-base.vue

```diff
@@ -0,0 +1,507 @@
+<template>
```

Comment:
This file looks really big... can you move each step into it's own file and wire them all together from this component?

Reply:
I will push up a new version of this branch ASAP.  The logic on this component now handles all aspects of rendering, validating, and committing to state the form questions from a single object  dynamically (including specifying minLength and numeric/alphanumeric validation) whereas before it would haver rendered the template dynamically but the getters and setters and computed values would still have had to be baked in.  result is a shorter more concise file that works better